connector-template/test.rs

```diff
@@ -22,70 +21,402 @@ impl utils::Connector for {{project-name | downcase | pascal_case}} {
     fn get_auth_token(&self) -> types::ConnectorAuthType {
         types::ConnectorAuthType::from(
             connector_auth::ConnectorAuthentication::new()
-                .{{project-name | downcase }}
+                .{{project-name | downcase}}
                 .expect("Missing connector authentication configuration"),
         )
     }
 
     fn get_name(&self) -> String {
-        "{{project-name | downcase }}".to_string()
+        "{{project-name | downcase}}".to_string()
     }
 }
 
+static CONNECTOR: {{project-name | downcase | pascal_case}} = {{project-name | downcase | pascal_case}} {};
+
+// Cards Positive Tests
+// This test creates a payment using the manual capture flow (Non 3DS).
 #[actix_web::test]
 async fn should_only_authorize_payment() {
-    let response = {{project-name | downcase | pascal_case}} {}.authorize_payment(None).await.unwrap();
-    assert_eq!(response.status, enums::AttemptStatus::Authorized);
+    let authorize_response = CONNECTOR.authorize_payment(None, None).await.unwrap();
+    assert_eq!(authorize_response.status, enums::AttemptStatus::Authorized);
 }
 
+// This test captures a payment using the manual capture flow (Non 3DS).
 #[actix_web::test]
-async fn should_authorize_and_capture_payment() {
-    let response = {{project-name | downcase | pascal_case}} {}.make_payment(None).await.unwrap();
-    assert_eq!(response.status, enums::AttemptStatus::Charged);
+async fn should_capture_authorized_payment() {
+    // Authorize
+    let authorize_response = CONNECTOR.authorize_payment(None, None).await.unwrap();
+    assert_eq!(authorize_response.status, enums::AttemptStatus::Authorized);
+    let txn_id = utils::get_connector_transaction_id(authorize_response);
+    assert_ne!(txn_id, None, "Empty connector transaction id");
+
+    // Capture
+    let capture_response = CONNECTOR
+        .capture_payment(txn_id.unwrap(), None, None)
+        .await
+        .unwrap();
+    assert_eq!(
+        capture_response.status,
+        enums::AttemptStatus::CaptureInitiated
+    );
 }
 
+// This test synchronizes a payment using the manual capture flow (Non 3DS).
 #[actix_web::test]
-async fn should_capture_already_authorized_payment() {
-    let connector = {{project-name | downcase | pascal_case}} {};
-    let authorize_response = connector.authorize_payment(None).await.unwrap();
+async fn should_synchronize_captured_payment() {
```

Comment:
```suggestion
async fn should_synchronize_authorized_payment() {
```

Reply:
@jagan-jaya This PR wasn't ready at the time you made this comment. I intentionally avoided "waiting of review" label, so that I can fix these known issues.